### PR TITLE
add getOffsets using adminClient

### PR DESCRIPTION
**Pull request by elakito**
Signed-off-by: Aki Yoshida <elakito@gmail.com>

I created this PR to implement getOffsets using the new adminConsumer of vertx-3.9.4.
This PR should replace https://github.com/strimzi/strimzi-kafka-bridge/pull/455.

This PR contains a change in pom.xml to upgrade vertx to 3.9.4. Please let me know if this vertx upgrade should go into a separate PR or commit.

### CODE REVIEW

File: src/main/resources/openapi.json [hunk 1]
```diff
@@ -1098,6 +1098,66 @@
...
+                        }
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "topicname",
+                    "in": "path",
+                    "description": "Name of the topic containing the partition.",
+
```
Comment (ppatierno) [inline src/main/resources/openapi.json:1144]: remove blank line

**Review by ppatierno — APPROVED** [review]
LGTM

**Review by stanlyDoge — APPROVED** [review]

File: documentation/book/api/paths.adoc [hunk 2]
```diff
@@ -1464,4 +1464,69 @@ __required__|Name of the topic to send records to or retrieve metadata from.|str
...
+__required__|Name of the topic containing the partition.|string
+|===
+
+
+==== Responses
+
+[options="header", cols=".^2a,.^14a,.^4a"]
+|===
+|HTTP Code|Description|Schema
+|**200**|a summary of the offsets|<<_offsetssummary,OffsetsSummary>>
```
Comment (PaulRMellor) [inline documentation/book/api/paths.adoc:1491]:
```suggestion
|**200**|A summary of the offsets|<<_offsetssummary,OffsetsSummary>>
```

File: src/main/resources/openapi.json [hunk 3]
```diff
@@ -1098,6 +1098,66 @@
...
+        "/topics/{topicname}/partitions/{partitionid}/offsets": {
+            "get": {
+                "tags": [
+                    "Topics"
+                ],
+                "description": "Retrieves a summary of the offsets for the topic partition.",
+                "operationId": "getOffsets",
+                "responses": {
+                    "200": {
+                        "description": "a summary of the offsets",
```
Comment (PaulRMellor) [inline src/main/resources/openapi.json:1110]:
```suggestion
                        "description": "A summary of the offsets.",
```

File: src/main/resources/openapiv2.json [hunk 4]
```diff
@@ -996,6 +996,60 @@
...
+          "Topics"
+        ],
+        "description": "Retrieves a summary of the offsets for the topic partition.",
+        "operationId": "getOffsets",
+        "produces": [
+          "application/vnd.kafka.v2+json"
+        ],
+        "responses": {
+          "200": {
+            "description": "a summary of the offsets",
```
Comment (PaulRMellor) [inline src/main/resources/openapiv2.json:1011]:
```suggestion
            "description": "A summary of the offsets.",
```

**Review by PaulRMellor — APPROVED** [review]
Couple of minor suggestions.

File: src/main/resources/openapi.json (hunk 1 above)
Comment (elakito) [reply src/main/resources/openapi.json:1144]: ok

File: documentation/book/api/paths.adoc (hunk 2 above)
Comment (elakito) [reply documentation/book/api/paths.adoc:1491]: ok

**Comment by elakito** [comment]
I have made the suggested changes.
thanks.

**Pull request merged**